contracts/exchangeV3/wrappersV3/MockWrapper.sol

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MIT
+pragma solidity =0.8.10;
+
+import "../../interfaces/exchange/IExchangeV3.sol";
+import "../../interfaces/curve/ISwaps.sol";
+import "../../interfaces/IERC20.sol";
+import "../../DS/DSMath.sol";
+import "../../auth/AdminAuth.sol";
+import "../../utils/SafeERC20.sol";
+import "../../utils/TokenUtils.sol";
+import "./helpers/WrapperHelper.sol";
+
+/// @title DFS exchange wrapper used for mocking in tests
+contract MockWrapper is DSMath, IExchangeV3, AdminAuth, WrapperHelper {
```

Comment:
Good idea to mock this, in a new push to main branch we now have `contracts/mocks` folder where we can put mock contracts to separate them from production contracts, shift there and rename the contract to `MockExchangeWrapper` to be more descriptive.